test/rb/test_rb.py

```diff
@@ -773,6 +773,92 @@ def test_gates_per_clifford_with_invalid_basis(self):
 
         self.assertAlmostEqual(gpc[0]['fake_gate'], 0)
 
+    def test_calculate_1q_epg(self):
+        """Test calculating EPGs of single qubit gates."""
+        gpc = {0: {'cx': 0, 'u1': 0.1, 'u2': 0.3, 'u3': 0.5}}
+        epc = 2.6e-4
+
+        epg_u1 = 0
+        epg_u2 = epc / (0.3 + 2 * 0.5)
+        epg_u3 = 2 * epg_u2
+
+        epgs = rb.calculate_1q_epg(gpc, epc, 0)
+
+        # test raise error when invalid qubit is specified.
+        with self.assertRaises(KeyError):
```

Comment:
If make the above changes to exceptions these should all check for `assertRaises(QiskitError):`